api/rest/client/methods_test.go

```diff
@@ -219,14 +219,40 @@ func TestStatusAll(t *testing.T) {
 	defer shutdown(api)
 
 	testF := func(t *testing.T, c Client) {
-		pins, err := c.StatusAll(false)
+		pins, err := c.StatusAll("", false)
 		if err != nil {
 			t.Fatal(err)
 		}
 
 		if len(pins) == 0 {
 			t.Error("there should be some pins")
 		}
+
```

Comment:
Ideally, tests like this should be structured as table-driven tests. But happy to leave this as is.